cot/src/auth/db.rs

```diff
@@ -7,6 +7,9 @@ use std::any::Any;
 use std::borrow::Cow;
 
 use async_trait::async_trait;
+// Importing `Auto` from `cot` instead of `crate` so that the migration generator
+// can figure out it's an autogenerated field
+use cot::db::Auto;
```

Comment:
Why can't it figure out if it's imported from `crate?

Reply:
It can, but it would require some additional logic in the symbol resolver that only benefits us only inside the `cot` crate. Might be worth adding, but it's definitely not a 5 minute change. I'll add a task so that we don't forget about it.

Reply:
#150